src/main/resources/hudson/plugins/plot/PlotReport/jump-to-plot.js

```diff
@@ -0,0 +1,8 @@
+document.addEventListener('DOMContentLoaded', function () {
+    const plotSelector = document.querySelector('.plot-selector');
+    if (plotSelector) {
```

Comment:
Isn't it safe to assume this always exists? I don't see any conditional rendering in the view.